src/components/review/ReviewsList.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import axiosInstance from '../../apis/axiosClient';
+
+const AlbumReviews = ({ albumId }) => {
+  const [reviews, setReviews] = useState([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    // Fetch reviews for the given album
+    async function fetchAlbumReviews() {
+      try {
+        const response = await axiosInstance.get(`/reviews/album/${albumId}`);
+        console.log(response.data.allProductReviews);
+
+        const { allProductReviews } = response.data;
+        setReviews(allProductReviews);
+        setLoading(false);
+      } catch (error) {
+        console.error('Error fetching album reviews:', error);
+        setLoading(false);
+      }
+    }
+
+    fetchAlbumReviews();
+  }, [albumId]);
+
+  return (
+    <div>
+      <h2>Album Reviews</h2>
+      {loading ? (
+        <p>Loading reviews...</p>
+      ) : (
+        <ul>
+          {reviews.map((review) => (
+            <li key={review._id}>
+              <h3>{review.title}</h3>
+              <p>Rating: {review.rating}</p>
+              <p>{review.comment}</p>
+            </li>
+          ))}
+        </ul>
+      )}
+    </div>
```

Comment:
Here is where we should render `WriteReview` component.

This is a better place for it because then we can also access any logic we need about reviews like:
- check if user is even logged in; if not, they should not see WriteReview component (but can still read reviews)
- check if any of the reviews we got were by this user, if so, they should not see the WriteReview component (because you are not allowed to write two reviews for the same comment) but instead can edit their review (can be implemented in a separate PR).